src/java.base/share/classes/jdk/internal/util/regex/Grapheme.java

```diff
@@ -267,7 +267,10 @@ static int getType(int cp) {
             if (cp >= 0xA960 && cp <= 0xA97C)
                 return L;
             //  hangul jamo_extended B
-            if (cp >= 0xD7B0 && cp <= 0xD7C6)
+            //  Kirat Rai vowel sign
+            if (cp >= 0xD7B0 && cp <= 0xD7C6 ||
+                cp == 0x16D63 ||
```

Comment:
LGTM as this matches up with https://www.unicode.org/reports/tr29/tr29-45.html#V and the `case 0x113D1:` lines up with _TULU-TIGALARI REPHA_.